Docs/Book/Install.md

```diff
@@ -67,6 +67,38 @@ Once "make" and "make install" completed successfully, use the following command
 This is required due to the [System Integrity Protection SIP](https://en.wikipedia.org/wiki/System_Integrity_Protection)
 introduced in more recent macOS versions.
 
+#### Linux x86_64: Python 3 environment
+
+The following commands will create a development environment for Linux x86_64 and Python 3.
+
+Start by downloading the latest anaconda installer from [Anaconda](https://www.anaconda.com/download/#linux) and install it. Then, install the required packages:
+
+	bash Anaconda3-5.2.0-x86_64.sh
+    conda install -y cmake cairo pillow eigen pkg-config
+    conda install -y boost-cpp boost py-boost
+
+Numpy and matplotlib are already part of the base installation of anaconda. Due to the latest boost libraries being currently built with a GLIBC version higher than the default in anaconda, we need to update to a more recent version:
+
+	conda install -y gxx_linux-64
+
+After that, numpy's include path must be fixed to allow RDKit to find its headers in the proper location. For that, from your anaconda root directory (usually ~/anaconda3/), run:
+
+	ln -s ./lib/python3.6/site-packages/numpy/core/include/numpy ./include/
+	
+At this point, you should be able to clone the RDKit repository to the desired build location, and start the build:
+
+	git clone https://github.com/rdkit/rdkit.git
+	cd rdkit
+	mkdir build && cd build
+	cmake .. -DPy_ENABLE_SHARED=1 \
+		-DRDK_INSTALL_INTREE=ON \
+		-DRDK_INSTALL_STATIC_LIBS=OFF \
+		-DRDK_BUILD_CPP_TESTS=ON \
+		-DRDK_OPTIMIZE_NATIVE=OFF
```

Comment:
Why not set optimize_native to on?

Reply:
My goal was to achieve a debug build that I can use for development, and forgot to update that for the build instructions.
I will remove the line to use the default value, which is 'ON'.